cmd/cloudx/project/patch_permission_config_test.go

```diff
@@ -6,35 +6,81 @@ package project_test
 import (
 	"testing"
 
-	"github.com/ory/cli/cmd/cloudx/testhelpers"
-
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/tidwall/gjson"
+
+	"github.com/ory/cli/cmd/cloudx/testhelpers"
 )
 
 func TestPatchPermissionConfig(t *testing.T) {
-	project := testhelpers.CreateProject(t, defaultConfig)
 	t.Run("is able to replace a key", func(t *testing.T) {
-		stdout, _, err := defaultCmd.Exec(nil, "patch", "keto-config", project, "--format", "json", "--add", `/namespaces=[{"name":"files", "id": 2}]`)
-		require.NoError(t, err)
-		assert.Equal(t, "files", gjson.Get(stdout, "namespaces.0.name").String(), stdout)
+		testhelpers.SetDefaultProject(t, defaultConfig, defaultProject)
+		t.Run("explicit project", func(t *testing.T) {
+			assert.Equal(t, defaultProject, testhelpers.GetDefaultProject(t, defaultConfig))
```

Comment:
Why do we need to assert this every time?

Reply:
I was hunting some flakes, but have now refactored the whole thing